linera-execution/src/wasm/mod.rs

```diff
@@ -97,7 +97,7 @@ impl WasmApplication {
 #[cfg(any(feature = "wasmer", feature = "wasmtime"))]
 #[derive(Debug, Error)]
 pub enum WasmExecutionError {
-    #[error("Failed to load WASM module")]
+    #[error("Failed to load WASM module: {_0}")]
```

Comment:
Did you mean `{0}`?

Reply:
No, `thiserror::Error` binds the `_0`, `_1`, ... names to the positional arguments in the tuple of the variant.